40196.0.txt

```diff
@@ -0,0 +1,65 @@
+@0
+MOV RIGHT,DOWN
+
+@1
+MOV 1,LEFT  # These instructions send two copies of dummy data for each node.
+MOV 1,RIGHT # This kick-starts the right and lower nodes
+MOV 1,DOWN  # The nodes with a longer delayed start hold more recent data
+MOV 1,LEFT  # The nodes which transfer the input immediately hold older data, which hangs around in the pipes as the final node waits for a value from everyone I guess???
+MOV 1,RIGHT # I don't even know how I worked this out
+MOV 1,DOWN  # How the %$&$ is this working
```

Comment:
Admittedly it's awkward in TIS-100, but this is why you should comment your code as you write it rather than afterwards!